modules/services/window-managers/i3-sway/lib/options.nix

```diff
@@ -7,14 +7,36 @@ let
   isI3 = moduleName == "i3";
   isSway = !isI3;
 
-  fonts = mkOption {
-    type = types.listOf types.str;
-    default = [ "monospace 8" ];
-    description = ''
-      Font list used for window titles. Only FreeType fonts are supported.
-      The order here is important (e.g. icons font should go before the one used for text).
-    '';
-    example = [ "FontAwesome 10" "Terminus 10" ];
+  fontOptions = types.submodule {
+    options = {
+      names = mkOption {
+        type = types.listOf types.str;
+        default = [ "monospace" ];
+        description = ''
+          List of font names list used for window titles. Only FreeType fonts are supported.
+          The order here is important (e.g. icons font should go before the one used for text).
+        '';
+        example = [ "FontAwesome" "Terminus" ];
+      };
+
+      style = mkOption {
+        type = types.str;
+        default = "";
+        description = ''
+          The font style to use for window titles.
+        '';
+        example = "Bold Semi-Condensed";
```

Comment:
where can I find these styles ? is this a pango property ?

Reply:
Yes, it's part of pango. Unfortunately, in my bit of searching, I can't find any reference to the actual set of styles that you can apply.